src/components/AnswersUpsert.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Control } from 'react-hook-form';
+import { TextFieldElement, useFieldArray } from 'react-hook-form-mui';
+import { Button, IconButton, Stack, Typography } from '@mui/material';
+import { Cancel } from '@mui/icons-material';
+import { useEffect, useState } from 'react';
+
+import type { RiddleUpsertDetail } from '../utils/Types.ts';
+
+type Props = {
+	control: Control<RiddleUpsertDetail>;
+	questionIndex: number;
+};
+
+export const AnswersUpsert = ({ control, questionIndex }: Props) => {
+	const minLength = 1;
```

Comment:
Opět detail. Toto jsem zvyklý dávat klidně i mimo komponentu a pojménovat kapitálkama, tedy např.: 
```
const MIN_ANSWERS_LENGTH = 1;
````
Ale určitě na tom netrvám.